library/nfsShare.py

```diff
@@ -47,10 +47,10 @@ def create(params, unity):
 
 def modify(params, unity):
     storageId = params.get('storageResource').get('id')
-    nfsShareId = params.get('nfsShare').get('id')
+    nfsShareId = params.get('nfsShare')
     # if not validator.check_parameters(params, parameters_all.get('modify')):
     # supportive_functions.raise_exception_about_results(parameters_all.get('modify'))
```

Comment:
remove comments

Reply:
my bad. It shouldn't be commented now, as Vadim has fixed the validator.